### PR TITLE
Update Docker compose example

### PR DESCRIPTION
**Pull request by chaudum**
to work with Loki 2.4.x

Signed-off-by: Christian Haudum <christian.haudum@gmail.com>

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:
Fixes #<issue number>

**Special notes for your reviewer**:

**Checklist**
- [ ] Documentation added
- [ ] Tests updated
- [ ] Add an entry in the `CHANGELOG.md` about the changes.

### CODE REVIEW

File: production/docker/docker-compose-ha-memberlist.yaml [hunk 1]
```diff
@@ -5,15 +5,30 @@ networks:
...
 
+  # Since the Loki containers are running as user 10001 and the mounted data volume is owned by root,
+  # Loki would not have permissions to create the directories.
+  # Therefore the init container changes permissions of the mounted directory.
+  init:
+    image: grafana/loki:2.4.1
+    user: root
+    entrypoint:
+      - "chown"
+      - "10001:10001"
```
Comment (jeschkies) [inline production/docker/docker-compose-ha-memberlist.yaml:16]: Why is this required?

**Review by cyriltovena — APPROVED** [review]
LGTM

**Comment by cyriltovena** [comment]
Not sure what's up with the CLA

**Review by kavirajk — APPROVED** [review]
LGTM

File: production/docker/docker-compose-ha-memberlist.yaml (hunk 1 above)
Comment (chaudum) [reply production/docker/docker-compose-ha-memberlist.yaml:16]: This is required because the mounted directories have the permissions of the host user.
